packages/react-core/src/demos/DataListDemo.md

```diff
@@ -15,6 +15,7 @@ import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/excl
 import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
 import TimesCircleIcon from '@patternfly/react-icons/dist/esm/icons/times-circle-icon';
 import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+import { rows, getRandomInteger } from '@patternfly/react-table/dist/esm/demos/sampleData';
```

Comment:
is this okay @tlabaj @thatblindgeye ? i know some projects are strict about using imports from other parts of the codebase like this?

functionally it works, just wondering if it should be moved or copied over somewhere so core/datalist doesn't depend on data from table.

Reply:
@gitdallas I agree, we should copy this data locally in react-core.